docs/src/main/asciidoc/deploying-native-executable-openshift.adoc

```diff
@@ -0,0 +1,218 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="deploy-openshift-native-executable"]
+= Deploy a Quarkus native application on OpenShift
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud,native
+
+Deploy native Quarkus applications to OpenShift Container Platform by using the Docker build strategy enabled by the Quarkus OpenShift extension.
+
+Quarkus supports the `quarkus-container-image-docker` extension for building container images.
```

Comment:
I think that this paragraph is not accurate.
The docker build strategy enabled by the openshift extension has nothing to do with `quarkus-container-image-docker` it's a feature provided by `quarkus-container-image-openshift` that is automatically brought into the project when users add `quarkus-openshift` to their project.

Reply:
Thank you. Deleted the line "Quarkus supports the `quarkus-container-image-docker` extension for building container images."